Public/Src/Engine/UnitTests/Scheduler/ArgumentsBuilder.cs

```diff
@@ -10,52 +10,221 @@
 
 namespace Test.BuildXL.Scheduler
 {
+    /// <summary>
+    /// Interface for building argument data.
+    /// </summary>
+    internal interface IArgumentsDataBuilder

```

Comment:
>IArgumentsDataBuilder [](start = 23, length = 21)

should this be in it's own file?

Reply:
When things become complex, I'll move it to a separate file. But this one is only used by unit tests. I don't feel like introducing too many new files if not necessary (e.g., for refactoring).

---
In reply to: [334140655](https://github.com/microsoft/BuildXL/pull/1034#discussion_r334140655) [](ancestors = 334140655)